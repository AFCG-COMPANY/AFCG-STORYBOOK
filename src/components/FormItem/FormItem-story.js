import React from 'react';
import { storiesOf } from '@storybook/react';
import FormItem from './FormItem';
import NumberInput from '../NumberInput';

storiesOf('FormItem', module).addWithInfo(
  'Default',
  `
    Form item.

  `,
  () => (
    <FormItem>
      <NumberInput id="number-input-1" />
    </FormItem>
  )
);
