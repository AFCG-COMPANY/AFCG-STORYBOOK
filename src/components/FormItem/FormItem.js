import PropTypes from 'prop-types';
import React from 'react';
import classnames from 'classnames';

const FormItem = ({ className, children, ...other }) => {
  const classNames = classnames('bx--form-item', className);

  return (
    <div className={classNames} {...other}>
      {children}
    </div>
  );
};

FormItem.propTypes = {
  children: PropTypes.node,
  className: PropTypes.string,
};

export default FormItem;
