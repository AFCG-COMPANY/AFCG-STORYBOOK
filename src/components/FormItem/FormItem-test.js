import React from 'react';
import { shallow } from 'enzyme';
import FormItem from '../FormItem';

describe('FormItem', () => {
  it('should render', () => {
    const wrapper = shallow(<FormItem />);
    expect(wrapper).toMatchSnapshot();
  });
});
