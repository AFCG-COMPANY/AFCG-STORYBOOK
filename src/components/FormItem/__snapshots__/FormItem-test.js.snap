// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormItem should render 1`] = `
<div
  className="bx--form-item"
/>
`;
